go/api/generic_commands.go

```diff
@@ -30,4 +30,300 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Exists returns the number of keys that exist in the database
+	//
+	// Note:
+	//When in cluster mode, the command may route to multiple nodes when keys map to different hash slots.
+	//
+	// Parameters:
+	// keys - One or more keys to delete.
+	//
+	// Return value:
+	// Returns the number of existing keys.
+	//
+	// Example:
+	// result, err := client.Exists([]string{"key1", "key2", "key3"})
+	//	if err != nil {
+	//	    // handle error
+	//	}
+	//	fmt.Println(result) // Output: 2
```

Comment:
`result.value` will be be 2 and `result.isNil` will be false. Please refer to documentation of `string_commands` and try to fix for all the commands in this file.

Reply:
done